includes/kvizzle.inc

```diff
@@ -97,17 +97,17 @@ const KVIZ_ACTION_VALUE = 33;		// :value
 const KVIZ_ACTION_VALUEORSECTION = 34; // :value-or-section
 
 
-enum KvizData {
-	Handle:Kviz_hKv,
-	Kviz_iPositionPathCount,
+enum struct KvizData {
```

Comment:
this had to be updated to an enum struct, as "old style" hacky enums were deprecated in sourcemod 1.11, which also explains the rest of this inc being updated to use proper methodmaps in the context of using the KvisData type